js/ui/overview.js

```diff
@@ -756,6 +757,9 @@ const Overview = new Lang.Class({
         this._coverPane.hide();
 
         this.emit('shown');
+        if (!this._shownOnce)
```

Comment:
It's just a nit, but I would probably not bother with the `if` clause and just always set the variable true.

Reply:
I thought the same, but in the end I decided to do it this way to make it more explicit that this variable is only expected to change its value once, since otherwise it might look like is one of those flags that should be set to one value in the `show()` function and then set to the opposite value in `showDone()`.

That said, a comment would suffice too, will change it